bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/presentation-download-dropdown/component.jsx

```diff
@@ -82,30 +84,57 @@ class PresentationDownloadDropdown extends PureComponent {
 
     this.menuItems = [];
 
-    const toggleDownloadOriginalPresentation = (enableDownload) => {
+    const { filenameConverted, filename, downloadableExtension } = item;
+    const convertedFileExtension = filenameConverted?.split('.').slice(-1)[0] || 'pdf';
```

Comment:
Looks like the converted file won't be always a `pdf`.
So I suggest to remove the default operator for this case.